addons/web/static/src/views/list/list_renderer.js

```diff
@@ -280,6 +280,7 @@ export class ListRenderer extends Component {
                     hasLabel: true,
                     label: propertyField.string,
                     sortable: false,
+                    attrs: { sum: propertyField.string },
```

Comment:
Shouldn't we specify it only for integer / float property fields?

(Even if it does not seem to break other types, a bit strange).

Reply:
yes we can :)